app/src/main/res/values/strings.xml

```diff
@@ -175,12 +175,13 @@
     <string name="past_positive_diagnoses">Past Positive Diagnoses</string>
     <string name="finish_code_verification_button">Finish Code Verification</string>
     <string name="call_button_text">Your positive diagnosis needs to be verified by a public health authority.</string>
-    <string name="call_button">Call 1–800–000-0000</string>
+    <string name="call_button">Call 1–800–000–0000</string>
```

Comment:
Mark this with a TODO that we need to change it + make an issue?

Reply:
It's too little for a separate issue imo.

Reply:
Oh...I misunderstood what you meant. It will suffice just to create a separate issue.